src/client/components/forms/work.js

```diff
@@ -112,6 +140,12 @@ class WorkForm extends React.Component {
 			});
 	}
 
+	/**
+	 * Renders the form component to allow the user to input
+	 * and submit data about a Work. Also shows a loading spinner
+	 * if connectivity causes a delay in submission.
+	 * @return {ReactElement} - react component (HTML) for a workeditor
```

Comment:
I'd change this to "React element corresponding to the rendered WorkEditor component"